File: crates/kernava-indexer/tests/fixtures/java-small/calc.java
package com.example;

import java.util.List;

public class Calculator {
    private int value;

    public Calculator(int v) {
        this.value = v;
    }

    public int add(int x, int y) {
        return compute(x);
    }

    private int helper(int x) {
        return x * 2;
    }

    public int compute(int x) {
        return helper(x) + value;
    }
}

interface Math {
    int compute(int x);
}

enum Color {
    RED, GREEN, BLUE
}

File: crates/kernava-indexer/tests/fixtures/cs-small/calc.cs
using System;
using Example.Math;

namespace Example {
    public class Calculator {
        private int value;

        public Calculator(int v) {
            value = v;
        }

        public int Add(int x, int y) {
            return Compute(x);
        }

        private int Helper(int x) {
            return x * 2;
        }

        public int Compute(int x) {
            return Helper(x) + value;
        }
    }

    public interface IMath {
        int Compute(int x);
    }

    public enum Color {
        Red, Green, Blue
    }
}
